client/package.json

```diff
@@ -2,14 +2,15 @@
   "dependencies": {
     "node-polyglot": "2.0.0",
     "normalize.css": "5.0.0",
-    "vue": "^2.1.0",
-    "vue-resource": "1.0.3",
-    "vue-router": "2.0.1"
+    "preact": "^6.4.0",
+    "preact-compat": "^3.9.2",
```

Comment:
For my culture, as we want to use directly `preact` here, we still need the `preact-compat` dependance?